src/TiledIteration.jl

```diff
@@ -4,6 +4,13 @@ using OffsetArrays
 using Base: tail, Indices, @propagate_inbounds
 using Base.IteratorsMD: inc
 
+if VERSION < v"1.6.0-DEV.1174"
```

Comment:
The current nightly is still v"1.6.0-DEV.1171" but I've tested it locally that it passes.